content/questions/actions/question-021.md

```diff
@@ -10,7 +10,7 @@ subject: []
 - [x] choice
 - [x] boolean
 - [x] string
-- [x] environment
-- [ ] number
+- [x] number
+- [ ] environment
```

Comment:
The [official documentation](https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputsinput_idtype) doesn't mention the existance of any `environment` input type.

Reply:
The current state of documentation is wrong then. It's very confusing but it's not for the very first time.

I've found the issue and raised a fix for the github documentation https://github.com/github/docs/pull/29781
The environment type was there but someone removed it and it got accepted :open_mouth: 

I'm sure this will get fixed, anyways **environment IS a valid input type 100%**